buku

```diff
@@ -4647,9 +4650,13 @@ def prompt(obj, results, noninteractive=False, deep=False, listtags=False, sugge
         skip_print = False
 
         try:
-            nav = read_in(PROMPTMSG)
+            prompt_suffix = ''
+            if bdb.dbfile != os.path.realpath(os.path.join(BukuDb.get_default_dbdir(), 'bookmarks.db')):
+                prompt_suffix = (f'[{bdb.dbname}] ' if not bdb.colorize else
+                                 f'\001\x1b[7\002m[{bdb.dbname}]\001\x1b[0m\002 ')
+            nav = read_in(PROMPTMSG + prompt_suffix)
```

Comment:
DB name suffix is calculated before printing out the prompt, so it's guaranteed to be the correct one at all times